### ISSUE TITLE
DeliciousError exception: urrlib.encode() should be encode()

### ISSUE THREAD

**Issue by GoogleCodeExporter** [peterhelin/pydelicious#23]
```
When throwing a DeliciousError exception, pydelicious calls
urllib.urlencode(). This should be just urlencode(), since it is imported
from urllib.
A patch is attached.
```

Original issue reported on code.google.com by `me.g...@gmail.com` on 23 Jan 2009 at 8:39

Attachments:
- [pydelicious.patch](https://storage.googleapis.com/google-code-attachments/pydelicious/issue-23/comment-0/pydelicious.patch)

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.5.2-rc2.
Added small unittest.
(changing status)
thnx.
```

Original comment by `berend.v...@gmail.com` on 2 Feb 2009 at 1:04
- Changed state: **Fixed**
